synapse/lib/httpapi.py

```diff
@@ -558,6 +558,9 @@ class AuthDelRoleV1(Handler):
 
 class ModelNormV1(Handler):
 
+    async def post(self):
+        await self.get()
+
     async def get(self):
 
         if not await self.reqAuthUser():
```

Comment:
Also, to prepare for the need to plumb this handler via a remote proxy, lets add a Cortex API and a CoreApi API for get PropNorm() so we can bridge this handler across a core that is really a telepath proxy